api/cpp/include/slint-platform.h

```diff
@@ -45,6 +45,53 @@ namespace slint {
 /// Another important class to subclass is the WindowAdapter.
 namespace platform {
 
+/// This struct contains getters that provide access to properties of the Window
+/// element, and is used with WindowAdapter::update_window_properties().
+struct WindowProperties
+{
+    SharedString title() const
+    {
+        SharedString out;
+        cbindgen_private::slint_window_properties_get_title(this, &out);
+        return out;
+    }
+
+    Brush background() const
+    {
+        Brush out;
+        cbindgen_private::slint_window_properties_get_background(this, &out);
+        return out;
+    }
+
+    /// The return value of WindowProperties::layout_constraints()
```

Comment:
Let's use the same description as in Rust. Are you sure about this being inside `WindowProperties`? (I don't mind, but was thinking that since it's in the platform module in Rust, maybe it should be in the platform namespace in C++?)

```suggestion
    /// /// This struct describes layout constraints of a resizable element, such as a window.
```

Reply:
The current documentation says it's a "return value". Sorry, that's not a sufficient description of what the data structure holds. The documentation should provide a brief overview of what the fields describe or what they're used for, right here, and not in some other place.

Reply:
I updated the docs